runtime-sdk/src/types/address.rs

```diff
@@ -241,4 +249,13 @@ mod test {
             Error::MalformedAddress
         ));
     }
+
+    #[test]
+    fn test_address_into_consensus_address() {
+        let pk = PublicKey::Secp256k1("Arra3R5V////////////////////////////////////".into());
```

Comment:
This seems like a bad example to test as the consensus layer doesn't support secp256k1 keys so such an address would be unusable.